src/web-component.js

```diff
@@ -48,6 +48,7 @@ class WebComponent extends HTMLElement {
       "with_projectbar",
       "project_name_editable",
       "with_sidebar",
+      "read_only",
```

Comment:
since this is a boolean I think it should also be added to the list on line 68 in `attributeChangedCallback()`, otherwise we might have some inconsistencies with falsy / truthy values

Reply:
Ah yes, I'd missed this, thanks, I'll update now 👍